06-classification_continued.Rmd

```diff
@@ -559,16 +657,378 @@ The overall workflow for performing K-nearest neighbour classification using `ti
 
 **Strengths:**
 
```

Comment:
I feel like we need a bit of a introductory sentence here, something like "All algorithms have strengths and weakenesses...."

Reply:
done